plugins/node/instrumentation-runtime-node/src/metrics/eventLoopUtilizationCollector.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { EventLoopUtilization, performance } from 'node:perf_hooks';
+import { RuntimeNodeInstrumentationConfig } from '../types';
+import { Meter } from '@opentelemetry/api';
+import { BaseCollector } from './baseCollector';
+
+const { eventLoopUtilization: eventLoopUtilizationCollector } = performance;
+
+export const NODEJS_EVENT_LOOP_UTILIZATION = 'eventloop.utilization';
+
+export class EventLoopUtilizationCollector extends BaseCollector {
+  constructor(
+    config: RuntimeNodeInstrumentationConfig = {},
+    namePrefix: string
+  ) {
+    super(config, namePrefix);
+  }
+
+  public updateMetricInstruments(meter: Meter): void {
+    meter
+      .createObservableGauge(
+        `${this.namePrefix}.${NODEJS_EVENT_LOOP_UTILIZATION}`,
+        {
+          description: 'Event loop utilization',
+          unit: 's',
+        }
+      )
+      .addCallback(async observableResult => {
+        if (!this._config.enabled) return;
+
+        const elu = eventLoopUtilizationCollector(this.scrape());
```

Comment:
this code is doing 2 consecutive calls to `eventLoopUtilizationCollector`
- 1st within `this.scrape` gets an "elu" object
- and just after that a second call with the 1st result

so the code is equivalent to
```js
const prev = eventLoopUtilizationCollector();
const elu = eventLoopUtilizationCollector(prev);
```

As a result the value of `elu.utilization` is always 1 which I think is wrong. Probably what we want is a diff between `elu` values taken with a time diff of `monitoringPrecision`. In that case is better to have a property with the last value and update in each callback (the scrape method is not needed anymore).

```js
.addCallback(async observableResult => {
        if (!this._config.enabled) return;

        const elu = eventLoopUtilizationCollector(this._lastValue);
        observableResult.observe(elu.utilization);
        this._lastValue = elu;
});
```

Reply:
fix